docs/UserGuide.md

```diff
@@ -7,14 +7,15 @@
 * **[3. Features](#features)**
   * [3.1 Adding an item: `ADD`](#adding-an-item-add)
   * [3.2 Editing an item: `EDIT`](#editing-an-item-edit)
-  * [3.3 Deleting an item: `Delete`](#deleting-an-item-del)
+  * [3.3 Deleting an item: `DEL`](#deleting-an-item-del)
   * [3.4 Marking an item as bought: `MARK`](#marking-an-item-as-bought-mark)
   * [3.5 Un-marking a marked item: `UNMARK`](#un-marking-a-marked-item-unmark)
   * [3.6 Listing all items: `DISPLAY`](#listing-all-items-display)
   * [3.7 Setting a budget: `SET`](#setting-a-budget-set)
-  * [3.8 Finding an item: `FIND`](#finding-an-item-find)
-  * [3.9 Viewing help: `HELP`](#viewing-help-help)
-  * [3.10 Exiting the Program: `BYE`](#exiting-the-program-bye)
+  * [3.8 Resetting a budget: `RES`](#resetting-a-budget-res)
+  * [3.9 Finding an item: `FIND`](#finding-an-item-find)
+  * [3.10 Viewing help: `HELP`](#viewing-help-help)
+  * [3.11 Exiting the Program: `BYE`](#exiting-the-program-bye)
 * **4. [FAQ](#faq)**
 * **5. [Command Summary](#command-summary)**
```

Comment:
I changed these lines in the PR I merged to make them consistent with the lines above. If it shows up in the merge conflict can help me keep those changes? Thanks!

Reply:
Yep noted, will keep your changes! Thanks :)